MvvmCross/Platforms/Uap/Presenters/MvxWindowsViewPresenter.cs

```diff
@@ -23,6 +23,9 @@ public class MvxWindowsViewPresenter
     {
         protected readonly IMvxWindowsFrame _rootFrame;
 
+        private MvxWindowsContentDialog mDialogView;
```

Comment:
Can we avoid by any chance holding a strong reference to the dialog view here?

Reply:
I don't know how to do it. You have access to the ViewModel, but not to the View.

Reply:
Doesn't follow the naming conventions we use.